install_config/configuring_authentication.adoc

```diff
@@ -1371,19 +1361,29 @@ additional query parameters):
    '<masterPublicURL>/oauth/authorize?client_id=openshift-challenging-client&response_type=token'
 ----
 
-. This should show a 401 response with a `WWW-Authenticate` basic challenge:
+. This should show a 401 response with a `WWW-Authenticate` basic challenge, a
+negotiate challenge, or both challenges:
 +
 ----
 #  curl -k -v -H 'X-Csrf-Token: 1' \
     '<redirected challengeURL from step 3 +query>'
 ----
 
-. This should show a redirect with an access token:
+. Confirm that you can log in to the `oc` command line using your Kerberos ticket:
+.. If you have not done so, provide your Kerberos credentials to get a ticket:
+----
+# kinit
+----
++
+Enter your Kerberos user name and password at the prompt. 
+.. Confirm that you can log in to the `oc` command line:
 +
 ----
-#  curl -k -v -u <your_user>:<your_password> \
-    -H 'X-Csrf-Token: 1' '<redirected_challengeURL_from_step_3 +query>'
+# oc login
 ----
++
+If your configuration is correct, you are logged in without entering separate
+credentials.
```

Comment:
There should add a different scenario, when no kerberos ticket issued or no krb5 workstation configured on the workstation, should fall back to basic authentication.

Reply:
Ok. @enj, do you have any objections to the new commit?